common/containers/Tabs/SendTransaction/components/ConfirmationModal.jsx

```diff
@@ -16,12 +18,14 @@ type Props = {
   rawTransaction: RawTransaction,
   wallet: BaseWallet,
   node: NodeConfig,
+  token: ?Token,
   onConfirm: RawTransaction => void,
   onCancel: () => void
 };
 
 type State = {
```

Comment:
Hypothetically it could, since it's calculated from props. I wouldn't want someone to introduce a bug where they reuse the same modal for multiple transactions, only changing the props, and not have it update accordingly.

Unless you're saying I should recalculate these every render, but that seems a little wasteful to me.